<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="choose_emot_1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>450</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Anger</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_button">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>590</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>541</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Qual emoção você identifica nesse vídeo?</string>
   </property>
  </widget>
  <widget class="QVideoWidget" name="video_player_1" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>256</width>
     <height>256</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>650</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Avançar</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="choose_emot_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>470</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fear</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="choose_emot_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>490</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Admiration</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="choose_video1_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>510</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Happy-for</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
